src/utils/fastLogin.ts

```diff
@@ -41,3 +41,14 @@ export const getIsUserElegibleForfastLogin = (
     (_fiscalCode) => false,
     FF_FastLogin
   );
+
+export const getLoginType = (
+  loginType: LoginTypeEnum | undefined,
+  isUserEligibleForFastLogin: boolean,
+  isLollipopEnabled: boolean
```

Comment:
What means that lollipop is enabled? This come from the feature flag relative to the lollipop feature. It can be removed in a near feature, seems to be irrealistic that lollipop can be disabled in production now.

Reply:
You're right, it's just that I wanted to revert to the pre-refactor situation for now and defer the removal of the feature flag to another task